src/oge/helpers.py

```diff
@@ -174,17 +202,11 @@ def create_plant_ba_table(year):
         plant_states, how="left", on="plant_id_eia", validate="m:1"
     )
 
-    # convert the dtype of the balancing authority code column from string to object
-    # this will allow for missing values to be filled
-    plant_ba["balancing_authority_code_eia"] = plant_ba[
-        "balancing_authority_code_eia"
-    ].astype(object)
-    plant_ba["balancing_authority_code_eia"] = plant_ba[
-        "balancing_authority_code_eia"
-    ].fillna(value=np.NaN)
```

Comment:
I'm assuming you removed this to have more consistent dtypes, but did you confirm that the original behavior is as expected? ie that missing BAs are getting filled properly?

Reply:
I did compare for 2022 the plant static attributes files before implementing the code in this PR and after:
```
>>> old[["plant_id_eia", "ba_code"]].equals(new[["plant_id_eia", "ba_code"]])
True
>>> old[["plant_id_eia", "ba_code", "state"]].compare(new[["plant_id_eia", "ba_code", "state"]])
      state      
       self other
797     NaN    WI
1924    NaN    VT
4319    NaN    MN
11630   NaN    MD
```
We can see that `ba_code` did not change between the two versions and that the 4 rows were `state` was missing are now filled out.

Reply:
sounds good